include/transport/time_queue.h

```diff
@@ -228,10 +228,10 @@ namespace qtransport {
 
         TimeQueue() = delete;
         TimeQueue(const TimeQueue&) = default;
-        TimeQueue(TimeQueue&&) = default;
+        TimeQueue(TimeQueue&&) noexcept = default;
```

Comment:
This and the below is [noexcept-move-constructor](https://clang.llvm.org/extra/clang-tidy/checks/performance/noexcept-move-constructor.html):

> Move constructors of all the types used with STL containers, for example, need to be declared noexcept. Otherwise STL will choose copy constructors instead. The same is valid for move assignment operations.